project/Dependencies.scala

```diff
@@ -61,7 +61,7 @@ object Dependencies {
     val protocJar = "com.github.os72" % "protoc-jar" % "3.11.4"
 
     val plexusBuildApi = "org.sonatype.plexus" % "plexus-build-api" % "0.0.7" % "optional" // Apache v2
-    val collectionCompat = "org.scala-lang.modules" %% "scala-collection-compat" % "2.5.0"
+    val collectionCompat = "org.scala-lang.modules" %% "scala-collection-compat" % "2.6.0"
```

Comment:
Do we have this dependency in other Akka satellite projects and could it cause some mismatch?

I don't think we have it in Akka core.

Reply:
> I don't think we have it in Akka core.

Correct, there we've cherry-picked the parts we need in our own internal `ccompat` since upstream didn't make binary compatibility promises back then, though it seems now they do (https://github.com/scala/scala-collection-compat/#usage) - so perhaps fine?